brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/ScribeReceiver.java

```diff
@@ -81,4 +77,11 @@ public ResultCode Log(final List<LogEntry> messages) throws TException {
         return result;
     }
 
+    public void setDelayMs(int delayMs) {
```

Comment:
I don't feel too strongly, just that setters are boilerplatey. If I were
writing this, I'd do it differently, but at this point we've given this
particular code far too much luxury of attention for its value. Let's move
on.